src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplateField/PhrasalTemplateRenderField.tsx

```diff
@@ -14,21 +14,26 @@ import {
   theme,
   variables,
 } from 'shared/styles';
+import { ItemResponseType } from 'shared/consts';
 
 import { StyledLineBreak } from './PhrasalTemplateField.styles';
 import { PhrasalTemplateFieldProps } from './PhrasalTemplateField.types';
-import { DisplayModeOptions } from './PhrasalTemplateField.const';
+import { DisplayModeOptions, KEYWORDS } from './PhrasalTemplateField.const';
 
 export function RenderedField({
   name = '',
   responseOptions = [],
-  type = 'sentence',
+  type = KEYWORDS.SENTENCE,
   ...otherProps
 }: Omit<PhrasalTemplateFieldProps, 'canRemove' | 'onRemove'>) {
   const [displayMode, setDisplayMode] = useState<{
     id: string;
     items?: Array<{ id: string; name: string }>;
   }>();
+  const [fromResponse, setFromResponse] = useState<{
```

Comment:
Nit: The dropdown has the title "Response From", not "From Response" – so I think it would be good to stay aligned in the identifier naming for consistency:
```suggestion
  const [responseFrom, setResponseFrom] = useState<{
```